pyproject.toml

```diff
@@ -29,11 +29,9 @@ classifiers = [
 "Bug tracker" = "https://github.com/ansible-community/antsibull-core/issues"
 
 [tool.poetry.dependencies]
-python = "^3.6.1"
-aiocontextvars = {version = "*", python = "~3.6"}
+python = "^3.9.0"
```

Comment:
Do we need a changelog fragment for bumping the minimum Python version?

Reply:
Oh, yes, definitely...

Reply:
d093c71b252c8fd7bf217b05dcd253e8b9069fc3.